src/db.js

```diff
@@ -0,0 +1,25 @@
+import { Sequelize } from 'sequelize';
+
+// const dbUser = process.env.POSTGRES_USER
```

Comment:
Don't push commented-out codes , always remember to get rid of them before shipping your codes